internal/request/graphql/schema/manager.go

```diff
@@ -188,6 +188,22 @@ func defaultTypes(
 	blobScalarType := schemaTypes.BlobScalarType()
 	jsonScalarType := schemaTypes.JSONScalarType()
 
+	idOpBlock := schemaTypes.IDOperatorBlock()
+	intOpBlock := schemaTypes.IntOperatorBlock()
+	floatOpBlock := schemaTypes.FloatOperatorBlock()
+	booleanOpBlock := schemaTypes.BooleanOperatorBlock()
+	stringOpBlock := schemaTypes.StringOperatorBlock()
+	jsonOpBlock := schemaTypes.JSONOperatorBlock(jsonScalarType)
+	blobOpBlock := schemaTypes.BlobOperatorBlock(blobScalarType)
+	dateTimeOpBlock := schemaTypes.DateTimeOperatorBlock()
+
+	notNullIntOpBlock := schemaTypes.NotNullIntOperatorBlock()
+	notNullFloatOpBlock := schemaTypes.NotNullFloatOperatorBlock()
+	notNullBooleanOpBlock := schemaTypes.NotNullBooleanOperatorBlock()
+	notNullStringOpBlock := schemaTypes.NotNullStringOperatorBlock()
+	notNullJSONOpBlock := schemaTypes.NotNullJSONOperatorBlock(jsonScalarType)
+	notNullBlobOpBlock := schemaTypes.NotNullBlobOperatorBlock(blobScalarType)
```

Comment:
question: I'm curious - why did you pull the existing stuff out of the return and into temp vars? And then add new func calls within the return?

Reply:
It made sense to me organizationally. Also the temp vars are used as inputs to the calls in the return.